From: Sven Eckelmann <sven@narfation.org>
Date: Tue, 14 Sep 2021 21:07:34 +0200
Subject: Revert "batman-adv: use Linux's stdarg.h"

This header is only available since Linux 5.15

This reverts commit 36d059797a14f0e373fdc3c79df7b467435925ad.

--- a/net/batman-adv/log.c
+++ b/net/batman-adv/log.c
@@ -7,7 +7,7 @@
 #include "log.h"
 #include "main.h"
 
-#include <linux/stdarg.h>
+#include <stdarg.h>
 
 #include "trace.h"
 
